src/SfxWeb/src/app/views/partition/backups/backups.component.ts

```diff
@@ -41,14 +52,88 @@ export class BackupsComponent extends PartitionBaseControllerDirective {
       new ListColumnSetting('raw.LsnOfLastBackupRecord', 'Lsn of last Backup Record'),
       new ListColumnSetting('raw.CreationTimeUtc', 'Creation Time UTC'),
     ]);
-    console.log(this);
+    this.debouncerHandlerSubscription = this.debounceHandler
+    .pipe(debounceTime(400), distinctUntilChanged())
+    .subscribe(dates => {
+        this.startDate = dates.startDate;
+        this.endDate = dates.endDate;
+        this.setNewPartitionBackupList(this.startDate, this.endDate);
+     });
+    this.dateRefresh = true;
+    this.minDate = new Date();
+    this.maxDate = new Date();
+  }
+
+  startTimeChange(){
+    const time = this.startTime.split(':');
+    this.startDate.setUTCHours(parseInt(time[0], 10), parseInt(time[1], 10), parseInt(time[2], 10), 0);
+    this.setNewPartitionBackupList(this.startDate, this.endDate);
+  }
+
+  endTimeChange(){
+    const time = this.endTime.split(':');
+    this.endDate.setUTCHours(parseInt(time[0], 10), parseInt(time[1], 10), parseInt(time[2], 10), 0);
+    this.setNewPartitionBackupList(this.startDate, this.endDate);
+  }
+
+  setNewPartitionBackupList(startDate: Date, endDate: Date)
+  {
+    this.backupList = this.partition.partitionBackupInfo.partitionBackupList.collection;
+    this.backupList = this.backupList.sort((left, right): number => {
+      if (left.CreationTimeUtc < right.CreationTimeUtc) {
+        return -1;
+      }
+      if (left.CreationTimeUtc > right.CreationTimeUtc) {
+        return 1;
+      }
+      return 0;
+    });
+    this.backupList = this.backupList.filter(function getData(info) {
```

Comment:
use an arrow function for consistency here

Reply:
Thanks Jeffrey, Made the necessary changes